app/views/records/show.html.erb

```diff
@@ -123,7 +123,7 @@
                 <!--Column 1: the text-->
 
                 <% unless review.review_comment.blank? %>
-                  <div class="comment_text_label" style="display: flex;" id=<%="text" + getFormReviewId(review)%>><%= string_html_format(review.review_comment) %></div>
+                  <div class="comment_text_label" style="display: flex; overflow-wrap: anywhere;" id=<%="text" + getFormReviewId(review)%>><%= string_html_format(review.review_comment) %></div>
```

Comment:
Suggest moving these inline styles into the comment_text_label class.

Reply:
Done, moved these inline styles into the comment_test_label class!